docs/io/output/tardis_tracking_example.yml

```diff
@@ -0,0 +1,57 @@
+# Example YAML configuration for TARDIS
```

Comment:
this feels like the standard example?

Reply:
This is not a standard example. There is changes to the number of packets so as to make it run fast for the tracking. More changes would be done in the future so as to make it very fast & not consume much time in testing.

Reply:
let's change this after